modules/ROOT/edits/getting-started-service-mesh.adoc

```diff
@@ -0,0 +1,17 @@
+= Get Started with Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, cluster, get started
+
+image::get-started-with-service-mesh-breadcrumb.png[75%, 75%]
+
+Use the information in this section to understand how to prepare, install, and use Anypoint Service Mesh.
```

Comment:
GLOBALLY replace all text references to sections, topics, this document, and so forth, per style guide (self-reference).
```suggestion
To get started with Anypoint Service Mesh, you must understand how to prepare, install, and use it.
```

Reply:
Done! Thanks!